magento/pwa-studio
# Mini Cart V2 / Shopping bag product listing

## Description

Added product listing component in the new Mini Cart component.

**Should be merged ahead of #2509 and #2511** 

## Related Issue
Closes [PWA-605](https://jira.corp.magento.com/browse/PWA-605)


### Verification Stakeholders
@jimbo 
@soumya-ashok 
@schensley 

### Verification Steps

1. run `yarn watch:venia`.
2. Add an item to the cart and click the new cart button.
3. We should be able to see the product listing.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/35203638/85473798-61850800-b579-11ea-8ffb-2a3c49e2bbc5.png)

## Checklist

* **Should be merged ahead of #2509 and #2511**
